### PR TITLE
Feature/integrate with hivemapper

### PR DESCRIPTION
**Pull request by BenDavidAaron**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -35,11 +39,39 @@
...
 
+# TODO: Rip out all Weights and Biases References
+# TODO: Rip out all GCP References
+# TODO: Rip out TensorBoard References?
+
 logger = logging.getLogger(__name__)
 
+# Get MLFlow basic auth credentials from AWS Secrets Store
+aws_secrets_client = boto3.client('secretsmanager', region_name="us-west-2")
+secrets = json.loads(aws_secrets_client.get_secret_value(SecretId="network/sandbox")["SecretString"])
```
Comment (evanmoss) [inline train.py:50]: move all environment input to input (env vars or program input), so `network/sandbox` and `us-west-2` should be input.

File: train.py [hunk 2]
```diff
@@ -35,11 +39,39 @@
 from utils.torch_utils import ModelEMA, select_device, intersect_dicts, torch_distributed_zero_first, is_parallel
 from utils.wandb_logging.wandb_utils import WandbLogger, check_wandb_resume
 
+# TODO: Rip out all Weights and Biases References
```
Comment (evanmoss) [inline train.py:42]: Maybe replace the TODOs with github issues

File: train.py [hunk 3]
```diff
@@ -699,4 +718,4 @@ def train(hyp, opt, device, tb_writer=None):
         # Plot results
         plot_evolution(yaml_file)
         print(f'Hyperparameter evolution complete. Best results saved as: {yaml_file}\n'
-              f'Command to train a new model with these hyperparameters: $ python train.py --hyp {yaml_file}')
+              f'Command to train a new model with these hyperparameters: $ python train.py --hyp {yaml_file}')
```
Comment (evanmoss) [inline train.py:721]: nit: newline

**Review by evanmoss — APPROVED** [review]
looks good after a couple edits

File: train.py (hunk 2 above)
Comment (BenDavidAaron) [reply train.py:42]: Done

File: train.py (hunk 3 above)
Comment (BenDavidAaron) [reply train.py:721]: `\n`'d

File: train.py (hunk 1 above)
Comment (BenDavidAaron) [reply train.py]: done

**Pull request merged**